### ISSUE TITLE
[terra-form-select] The size has been increased significantly since version 6.x.x 

### ISSUE THREAD

**Issue by yhatcerner** [cerner/terra-core#3435]
# Bug Report

## Description

Some of our dependencies are pulling terra-form-select which has a significant size increase since version 6.x.x, so it also increases the size of our project after the dependency updates. 

Size Reference of terra-form-select: https://bundlephobia.com/result?p=terra-form-select@5.45.0

## Steps to Reproduce

1. Install webpack-bundle-analyzer: https://www.npmjs.com/package/webpack-bundle-analyzer
2. Run 'npm run build:development' and you should be able to see a generated graph of the dependency size in the web browser

## Additional Context / Screenshots

![image](https://user-images.githubusercontent.com/83730820/117209216-adb8ef80-adbb-11eb-9f42-cb52612d2e53.png)
## Expected Behavior

As you can see in the above screenshot, Frame.js and Menu.js could be reused which could decrease the size.
## Possible Solution

There should be different versions of Frame.js and Menu.js used in the dependencies of terra-form-select. Unifying the version should help to fix the size issue.

## Environment

* Component Name and Version:
* Browser Name and Version:
* Node/npm Version: [e.g. Node 8/npm 5] Node^8.10.0 || ^10.13.0/npm 6.14.5
* Webpack Version: 4.44.1
* Operating System and version (desktop or mobile): macOS Catalina@10.15.7

## @ Mentions

n/a
